hypothesis/lms
# Fix grading comments in Blackboard

This bug is documented in "behind the blackboard case" `05333239`.


### Testing

- Remember, you'll need a working HTTPS localhost setup, see: https://github.com/hypothesis/lms#using-self-signed-certificates-with-https

- Enable grading comments on tha LTI1.3 BB AI: http://localhost:8001/admin/instance/100/


- Login as `blackboardteacher` in https://aunltd-test.blackboard.com/
- Launch the [LTI 1.3 assignment test assignment](https://aunltd-test.blackboard.com/webapps/blackboard/content/contentWrapper.jsp?content_id=_2085_1&displayName=localhost+PDF+-+LTI1.3&course_id=_19_1&navItem=content&href=%2Fwebapps%2Fblackboard%2Fexecute%2Fblti%2FlaunchPlacement%3Fblti_placement_id%3D_148_1%26content_id%3D_2085_1%26course_id%3D_19_1)

- Pick "Blackboard student" from the dropdown, the score will get updated and the comment will be now in the text area.


- Submit a new score and comment
- Refresh the assignment, pick the student again. The grade & comment wil be updated.
